src/components/FilterCheckbox/FilterCheckbox.tsx

```diff
@@ -0,0 +1,35 @@
+﻿import React from 'react';
+import CheckboxIcon from '../../icons/CheckboxIcon';
+import { FilterGroup, InputProps, JobType } from '../../types/types';
+
+interface CheckboxProps extends InputProps {
```

Comment:
**nitpick:** Should be generic instead of hardcoding specific use case:

```ts
interface CheckboxProps<T, U> extends InputProps {
    onChange: (field: T, value: U) => void;
    isChecked: boolean
}
```